GeneticAlgorithm.py

```diff
@@ -371,6 +385,29 @@ def initialize(self):
 
             self._assignments.append(newCourseAssignment)
 
+            if (len(GAList) - 1) == gata_lab_counter:
+                gata_lab_counter = 0
+            else:
+                gata_lab_counter += 1
+
+        for cur_lab in unique_labs:
+
+            gata = GAList[gata_course_counter]
+            newCourseAssignment = CourseAssignment(self._classNumb, gata)
+            self._classNumb += 1
+            # Setting the course, meeting time, and semester year and append it to newClass.
+            newCourseAssignment.set_course(cur_lab)
+            newCourseAssignment.set_meetingTime(cur_lab.get_meetTimes())
```

Comment:
This line causes an error: AttributeError: 'str' object has no attribute 'get_meetTimes'